apis/server.d.ts

```diff
@@ -6,7 +6,8 @@ import { Application, RequestHandler } from 'express'
 import { XOR } from './internal/util'
 
 type _cds = typeof cds
-type cds_services = { [name: string]: Service }
+type service_dict = { [name: string]: Service };
+interface cds_services extends service_dict {}
```

Comment:
let's actually just go all the way with this and replace the `type` with an `interface`, but please also have some comment of sorts to explain this change, i.e.

```ts
// interface instead of type so users can insert their actual Services via module augmentation
```